test/plugins/inlineStyles.07.svg

```diff
@@ -0,0 +1,17 @@
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <style>
+        .red {
+            fill: red !important;
+        }
+        .blue {
+            fill: blue;
+        }
+    </style>
+    <rect width="100" height="100" class="blue red"/>
+</svg>
+
+@@@
+
+<svg id="test" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+    <rect width="100" height="100" class="blue red" style="fill:red!important"/>
```

Comment:
Indeed, the [convertStyleToAttrs](https://github.com/svg/svgo/blob/master/plugins/convertStyleToAttrs.js) plugin.